### PR TITLE
export MZ API for windows

### PR DESCRIPTION
**Pull request by tsaizhenling**
**Describe the pull request**

- #### What does your PR fix?  
  fixes unresolved symbols on windows

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
I believe so

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/minizip-ng/portfile.cmake [hunk 1]
```diff
@@ -14,6 +14,8 @@ vcpkg_cmake_configure(
     DISABLE_PARALLEL_CONFIGURE
     OPTIONS 
         -DMZ_PROJECT_SUFFIX:STRING=-ng
+	-DMZ_EXPORTS=1
+	-DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON
```
Comment (Hoikas) [inline ports/minizip-ng/portfile.cmake:18]: https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#do-not-add-cmake_windows_export_all_symbols

Comment (JonLiu1993) [inline ports/minizip-ng/portfile.cmake:18]: CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS is a highly problematic behavior for many reasons, therefore our maintainer guide explicitly forbids adding it to libraries: https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#do-not-add-cmake_windows_export_all_symbols.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]
@tsaizhenling ,Can you please use command 'file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGE_DIRS}/share/${PORT} RENAME copyright)' to handle copyright?

File: ports/minizip-ng/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "minizip-ng",
-  "version": "3.0.2",
+  "version": "3.0.2.1",
```
Comment (JonLiu1993) [inline ports/minizip-ng/vcpkg.json:3]: We recommend adding'port-version: 1'to indicate the modified version

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@tsaizhenling ,Thanks for your pr,
after  modify the requested changes,please run the command './vcpkg x-add-version minizip-ng --overwrite-version' to commit the changes again

**Comment by JonLiu1993** [comment]
@tsaizhenling ,Could you modify the suggested content?

**Comment by tsaizhenling** [comment]
hi, thanks for the review, but since I'm have not been able to get attention from maintainer of minizip-ng on this, I will just be closing this, and unfortunately, I will be adding the minizip-ng dependency to my project the old fashioned way

**Pull request closed without merging**